src/main/java/gwt/jsonix/marshallers/xjc/plugin/builders/MainJsBuilder.java

```diff
@@ -158,7 +167,8 @@ protected static void addUnmarshall(JCodeModel toPopulate, JDefinedClass mainJsC
                           firstParameterName, secondParameterRef, secondParameterName, callbackRef);
     }
 
-    protected static void addMarshall(JCodeModel toPopulate, JDefinedClass mainJsClass, JClass firstParameterRef, JClass callbackRef) {
+    protected static void addMarshall(JCodeModel toPopulate, JDefinedClass mainJsClass, JClass
+            firstParameterRef, JClass callbackRef) {
```

Comment:
As we touched the header, could we use the format as we use for `generateJSInteropMainJs` please?

Reply:
@jomarko 
done